.github/workflows/screenshots.yml

```
@@ -41,7 +41,7 @@ jobs:
         uses: ./.github/workflows/actions/build
 
       - name: Start review application
-        run: npm run serve &
+        run: npm start --workspace app &
```

Need to make sure we skip the `prestart` build here (as it's cached in GitHub Actions)

```suggestion

        # Start Express.js without "prestart" build script
        run: npm start --ignore-scripts --workspace app &
```